Classes/Public/Manager/Submanagers/OCTSubmanagerCalls.h

```diff
@@ -34,6 +34,11 @@
 @property (weak, nonatomic) id<OCTSubmanagerCallDelegate> delegate;
 
 /**
+ * Set the property to YES to enable the microphone, otherwise YES.
+ **/
+@property (nonatomic, assign) BOOL enableMicrophone;
```

Comment:
I think it makes sense to make default value of `enableMicrophone` to `YES`. You can do that and add a comment about that here, as well as in engine documentation.